alvr/server_core/src/statistics.rs

```diff
@@ -271,13 +271,14 @@ impl StatisticsManager {
                 self.packets_lost_partial_sum = 0;
             }
 
-            // While not accurate, this prevents NaNs and zeros that would cause a crash or pollute
-            // the graph
-            let bitrate_bps = if network_latency != Duration::ZERO {
-                frame.video_packet_bytes as f32 * 8.0 / network_latency.as_secs_f32()
-            } else {
-                0.0
-            };
+            let packet_bits = frame.video_packet_bytes as f32 * 8.0;
+            let throughput_bps = packet_bits
+                / Duration::max(network_latency, Duration::from_millis(1)).as_secs_f32();
+            let bitrate_bps = packet_bits
+                / self
+                    .last_frame_present_interval
+                    .max(Duration::from_millis(1))
```

Comment:
And this should maybe be higher because even 120hz is 6 ms and accounting for fast frames it 3ms might be reasonable (but ig futureproofing would also be a consideration)

Reply:
Following the original purpose of the code, I've also made this `from_micros(1)`